compose-files/docker-compose.yml

```diff
@@ -162,6 +162,54 @@ services:
       - storage
       - default
     restart: unless-stopped
+  server-management:
+    image: ghcr.io/hpi-schul-cloud/schulcloud-server:${SERVER_DOCKER_TAG}
+    container_name: schulcloud-server-management
+    build:
+      context: ${SERVER_PATH}
+      dockerfile: Dockerfile
+      # cache_from:
+      #   - schulcloud/schulcloud-server:latest
+    depends_on:
+      - mongosetup
+      - redis
+      - rabbit
+      - calendar
+    command: npm run nest:start:management
+    environment:
+      # ENV
+      - NODE_ENV=${NODE_ENV:-development}
+      - HOST=http://localhost:3333
+      - SC_TITLE=HPI Schulcloud
+      - SC_SHORT_TITLE=Schulcloud
+      # URIs and Port
+      - DB_URL=mongodb://mongodb:27017,mongodb-secondary:27017,mongodb-arbiter:27017/schulcloud?replicaSet=rs0&tls=false&readPreference=secondary
+      - DATABASE__URL=mongodb://server-mongodb:27017/schulcloud
+      - REDIS_URI=redis://redis:6379
+      - RABBITMQ_URI=amqp://guest:guest@rabbit:5672
+      - CONTENT_URI=https://consent.schul-cloud.org #lernstore
+      - PORT=3333 # will be 3333 anyways
+      # AUTH Secret
+      # - AUTHENTICATION=
+      - IGNORE_DISCOVERABILITY=true
+      # Brute force protection
+      - LOGIN_BLOCK_TIME=3
+      # Edu-Sharing
+      - ES_DOMAIN=https://mv-repo.schul-cloud.org
+      - ES_USER={ES_USER}
+      - ES_PASSWORD={ES_PASSWORD}
+      - SECRET_ES_MERLIN_USERNAME={SECRET_ES_MERLIN_USERNAME}
+      - SECRET_ES_MERLIN_PW={SECRET_ES_MERLIN_PW}
+      # FEATURE TOGGLES
+      - LERNSTORE_MODE=EDUSHARING
+      - CALENDAR_URI=http://calendar:3000
+      - CALENDAR_API_KEY=test123
+    ports:
+      - "3333:3333"
+    networks:
+      - storage
+      - default
+    restart: unless-stopped
 
```

Comment:
it is possible instead of duplicating the code, to extend
https://stackoverflow.com/questions/45279662/extend-service-in-docker-compose-3